src/test/resources/jsonld/HT018857620

```diff
@@ -0,0 +1,168 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "containsExampleOfWork" : [ {
+    "creatorOfWork" : "Wagner, Richard 1813-1883",
+    "id" : "http://d-nb.info/gnd/300169892",
+    "label" : "Götterdämmerung",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Wagner, Richard 1813-1883",
+    "id" : "http://d-nb.info/gnd/300170114",
+    "label" : "Die Meistersinger von Nürnberg",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Berlioz, Hector 1803-1869",
+    "id" : "http://d-nb.info/gnd/30001922X",
+    "label" : "La damnation de Faust",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Verdi, Giuseppe 1813-1901",
+    "id" : "http://d-nb.info/gnd/300165633",
+    "label" : "La forza del destino",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Schubert, Franz 1797-1828",
+    "id" : "http://d-nb.info/gnd/300141777",
+    "label" : "Sinfonien",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Verdi, Giuseppe ",
+    "id" : "http://d-nb.info/gnd/1081395184",
+    "label" : "Don Carlos <Io I'ho perduta>",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Wagner, Richard 1813-1883",
+    "id" : "http://d-nb.info/gnd/300170602",
+    "label" : "Der Ring des Nibelungen",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Berlioz, Hector ",
+    "id" : "http://d-nb.info/gnd/1081431687",
+    "label" : "La damnation de Faust",
+    "type" : [ "Work" ]
+  }, {
+    "creatorOfWork" : "Wagner, Richard 1813-1883",
+    "id" : "http://d-nb.info/gnd/300169914",
```

Comment:
This looks very similar to the one above but it has a different GND ID. To make the difference between those clear, we will also have to add the sub title from sub field `u`, here  `<subfield code="u">Trauermarsch</subfield>`. See also https://github.com/hbz/lobid-resources/issues/556#issue-253656189.